index.html

```diff
@@ -22,206 +22,216 @@
+      <p>
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ut excepturi
+        voluptates quidem culpa pariatur, mollitia consectetur nemo esse
+        voluptate, modi ullam labore deleniti! At dolore exercitationem a, illo
+        porro perspiciatis.
+      </p>
 
-        <ul class="languages-list">
-          <h4 class="skill-subtitle">Languages</h4>
+      <button type="button" class="cv">
+        Get my CV <img src="images/ic_download.svg" alt="download-icon" />
+      </button>
+      <section class="languages-list">
+        <h4 class="skill-subtitle">Languages</h4>
+        <ul>
           <li>JavaScript (ES5/ES6)</li>
           <li>Ruby</li>
           <li>HTML</li>
           <li>CSS</li>
         </ul>
-
-        <ul class="frameworks-list">
-          <h4 class="skill-subtitle">Frameworks</h4>
+      </section>
+      <section class="frameworks-list">
+        <h4 class="skill-subtitle">Frameworks</h4>
+        <ul>
           <li>Bootstrap</li>
           <li>Ruby on rails</li>
           <li>RSpec</li>
           <li>Capibara</li>
           <li>Selenium</li>
         </ul>
-
-        <ul class="skills-list">
-          <h4 class="skill-subtitle">Skills</h4>
+      </section>
+      <section class="skills-list">
+        <h4 class="skill-subtitle">Skills</h4>
+        <ul>
           <li>Codekit</li>
           <li>Codepen</li>
           <li>GitHub</li>
           <li>GitLAb</li>
           <li>Terminal</li>
         </ul>
       </section>
-      <section class="get-started">
-        <h2 class="subtitle">Get Started</h2>
-        <p>Lorem ipsum dolor sit amet, consectetur 
-          adipisicing elit. Suscipit fugiat quam consectetur. 
-          Distinctio officiis dolorem excepturi natus quod 
-          perferendis, explicabo, omnis provident deserunt 
-          laudantium odit earum commodi, neque dolor. Dolorum.</p>
-      
+    </section>
+    <section class="get-started">
+      <h2 class="subtitle">Get Started</h2>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Suscipit
+        fugiat quam consectetur. Distinctio officiis dolorem excepturi natus
+        quod perferendis, explicabo, omnis provident deserunt laudantium odit
+        earum commodi, neque dolor. Dolorum.
+      </p>
+
       <form action="https://formspree.io/f/xoqyzbvj" method="post" class="form">
-      <ul>
-        <li>
-          <label for="name">Your Name</label>
-          <input type="text" id="name" name="name" placeholder="Name" maxlength="30" required>
-        </li>
-        <li>
-          <label for="email">Your Email</label>
-          <input type="email" id="email" name="email" placeholder="Email" required>
-        </li>
-        <li>
-          <textarea placeholder="Enter your message here..." maxlength="500" required></textarea>
-        </li>
-        <li>
-          <button type="submit">Start collaboration</button>
-        </li>
-      </ul>
+        <ul>
```

Comment:
[OPTIONAL] I think you missed the default styles of your lists in the mobile version while working on your desktop. You must have a `list-style-type` somewhere in your media query. Probably just moving out of there will do the trick.

![image](https://user-images.githubusercontent.com/79114233/139560573-4c2f6bae-a815-406f-ab13-a2976207da4d.png)

Reply:
Will fix this too!